### ISSUE TITLE
Test failure(s): AbstractWatchServiceTest

### ISSUE THREAD

**Issue by tomhoefer** [eclipse/smarthome#1753]
After #1597 has been merged the ``ÀbstractWatchServiceTest```s are failing on my macbook...

@dstivanov can you pls check this?

```
Tests run: 7, Failures: 5, Errors: 0, Skipped: 0, Time elapsed: 24.374 sec <<< FAILURE! - in org.eclipse.smarthome.core.service.AbstractWatchServiceTest
AbstractWatchQueueReader processWatchEvent path in subsubdir provided with correct arguments(org.eclipse.smarthome.core.service.AbstractWatchServiceTest)  Time elapsed: 3.073 sec  <<< FAILURE!
java.lang.AssertionError: At least one watch event is expected within 3 seconds.
Expected: is <true>
     but: was <false>
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.junit.Assert.assertThat(Assert.java:865)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:90)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:324)
	at org.codehaus.groovy.runtime.callsite.StaticMetaMethodSite$StaticMetaMethodSiteNoUnwrap.invoke(StaticMetaMethodSite.java:130)
	at org.codehaus.groovy.runtime.callsite.StaticMetaMethodSite.callStatic(StaticMetaMethodSite.java:99)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callStatic(AbstractCallSite.java:173)
	at org.eclipse.smarthome.core.service.AbstractWatchServiceTest$_fullEventAssertionsByKind_closure4.doCall(AbstractWatchServiceTest.groovy:258)
	at org.eclipse.smarthome.core.service.AbstractWatchServiceTest$_fullEventAssertionsByKind_closure4.doCall(AbstractWatchServiceTest.groovy)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:90)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:324)
	at org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:278)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1016)
	at org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.call(PogoMetaClassSite.java:39)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:108)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:112)
	at org.eclipse.smarthome.test.OSGiTest.waitForAssert(OSGiTest.groovy:181)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:207)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:56)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:49)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:133)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:149)
	at org.eclipse.smarthome.core.service.AbstractWatchServiceTest.fullEventAssertionsByKind(AbstractWatchServiceTest.groovy:258)

directory changes are watched correctly(org.eclipse.smarthome.core.service.AbstractWatchServiceTest)  Time elapsed: 3.02 sec  <<< FAILURE!
java.lang.AssertionError: Exactly 1 watch events were expected within 3 seconds, but were: []
Expected: is <1>
     but: was <0>
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.junit.Assert.assertThat(Assert.java:865)
	at org.junit.Assert$assertThat$1.callStatic(Unknown Source)
	at org.eclipse.smarthome.core.service.AbstractWatchServiceTest$_directory_changes_are_watched_correctly_closure3.doCall(AbstractWatchServiceTest.groovy:151)
	at org.eclipse.smarthome.core.service.AbstractWatchServiceTest$_directory_changes_are_watched_correctly_closure3.doCall(AbstractWatchServiceTest.groovy)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:90)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:324)
	at org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:278)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1016)
	at org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.call(PogoMetaClassSite.java:39)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.call(PogoMetaClassSite.java:54)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:112)
	at org.eclipse.smarthome.test.OSGiTest.waitForAssert(OSGiTest.groovy:181)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:207)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:56)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:49)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:133)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:149)
	at org.eclipse.smarthome.core.service.AbstractWatchServiceTest.directory changes are watched correctly(AbstractWatchServiceTest.groovy:151)

AbstractWatchQueueReader processWatchEvent path in root provided with correct arguments(org.eclipse.smarthome.core.service.AbstractWatchServiceTest)  Time elapsed: 3.012 sec  <<< FAILURE!
java.lang.AssertionError: At least one watch event is expected within 3 seconds.
Expected: is <true>
     but: was <false>
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.junit.Assert.assertThat(Assert.java:865)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:90)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:324)
	at org.codehaus.groovy.runtime.callsite.StaticMetaMethodSite$StaticMetaMethodSiteNoUnwrap.invoke(StaticMetaMethodSite.java:130)
	at org.codehaus.groovy.runtime.callsite.StaticMetaMethodSite.callStatic(StaticMetaMethodSite.java:99)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callStatic(AbstractCallSite.java:173)
	at org.eclipse.smarthome.core.service.AbstractWatchServiceTest$_fullEventAssertionsByKind_closure4.doCall(AbstractWatchServiceTest.groovy:258)
	at org.eclipse.smarthome.core.service.AbstractWatchServiceTest$_fullEventAssertionsByKind_closure4.doCall(AbstractWatchServiceTest.groovy)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:90)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:324)
	at org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:278)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1016)
	at org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.call(PogoMetaClassSite.java:39)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.call(PogoMetaClassSite.java:54)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:112)
	at org.eclipse.smarthome.test.OSGiTest.waitForAssert(OSGiTest.groovy:181)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:207)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:56)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:149)
	at org.eclipse.smarthome.core.service.AbstractWatchServiceTest.fullEventAssertionsByKind(AbstractWatchServiceTest.groovy:258)

same file names in root and subdir are correctly processed(org.eclipse.smarthome.core.service.AbstractWatchServiceTest)  Time elapsed: 3.216 sec  <<< FAILURE!
java.lang.AssertionError: Exactly two watch events were expected within 3 seconds, but were: []
Expected: is <2>
     but: was <0>
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.junit.Assert.assertThat(Assert.java:865)
	at org.junit.Assert$assertThat$1.callStatic(Unknown Source)
	at org.eclipse.smarthome.core.service.AbstractWatchServiceTest$_same_file_names_in_root_and_subdir_are_correctly_processed_closure2.doCall(AbstractWatchServiceTest.groovy:119)
	at org.eclipse.smarthome.core.service.AbstractWatchServiceTest$_same_file_names_in_root_and_subdir_are_correctly_processed_closure2.doCall(AbstractWatchServiceTest.groovy)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:90)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:324)
	at org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:278)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1016)
	at org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.call(PogoMetaClassSite.java:39)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.call(PogoMetaClassSite.java:54)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:112)
	at org.eclipse.smarthome.test.OSGiTest.waitForAssert(OSGiTest.groovy:181)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:207)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:56)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:49)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:133)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:149)
	at org.eclipse.smarthome.core.service.AbstractWatchServiceTest.same file names in root and subdir are correctly processed(AbstractWatchServiceTest.groovy:119)

subdirs are not registered and not watched(org.eclipse.smarthome.core.service.AbstractWatchServiceTest)  Time elapsed: 3.008 sec
subdirs are not watched, but dirs modifications are watched(org.eclipse.smarthome.core.service.AbstractWatchServiceTest)  Time elapsed: 6.005 sec
AbstractWatchQueueReader processWatchEvent path in subdir provided with correct arguments(org.eclipse.smarthome.core.service.AbstractWatchServiceTest)  Time elapsed: 3.02 sec  <<< FAILURE!
java.lang.AssertionError: At least one watch event is expected within 3 seconds.
Expected: is <true>
     but: was <false>
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.junit.Assert.assertThat(Assert.java:865)
	at sun.reflect.GeneratedMethodAccessor54.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:90)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:324)
	at org.codehaus.groovy.runtime.callsite.StaticMetaMethodSite$StaticMetaMethodSiteNoUnwrap.invoke(StaticMetaMethodSite.java:130)
	at org.codehaus.groovy.runtime.callsite.StaticMetaMethodSite.callStatic(StaticMetaMethodSite.java:99)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callStatic(AbstractCallSite.java:173)
	at org.eclipse.smarthome.core.service.AbstractWatchServiceTest$_fullEventAssertionsByKind_closure4.doCall(AbstractWatchServiceTest.groovy:258)
	at org.eclipse.smarthome.core.service.AbstractWatchServiceTest$_fullEventAssertionsByKind_closure4.doCall(AbstractWatchServiceTest.groovy)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:90)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:324)
	at org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:278)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1016)
	at org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.call(PogoMetaClassSite.java:39)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.call(PogoMetaClassSite.java:54)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:112)
	at org.eclipse.smarthome.test.OSGiTest.waitForAssert(OSGiTest.groovy:181)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:207)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:56)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:149)
	at org.eclipse.smarthome.core.service.AbstractWatchServiceTest.fullEventAssertionsByKind(AbstractWatchServiceTest.groovy:258)

Running org.eclipse.smarthome.core.storage.ManagedItemProviderOSGiTest
Tests run: 8, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.052 sec - in org.eclipse.smarthome.core.storage.ManagedItemProviderOSGiTest
assert two items with same name can not be added(org.eclipse.smarthome.core.storage.ManagedItemProviderOSGiTest)  Time elapsed: 0.011 sec
assert items are there once the factory gets added(org.eclipse.smarthome.core.storage.ManagedItemProviderOSGiTest)  Time elapsed: 0.014 sec
assert items are added as group members on deferred creation(org.eclipse.smarthome.core.storage.ManagedItemProviderOSGiTest)  Time elapsed: 0.007 sec
assert getItems returns item from registered ManagedItemProvider(org.eclipse.smarthome.core.storage.ManagedItemProviderOSGiTest)  Time elapsed: 0.003 sec
assert removal returns old value(org.eclipse.smarthome.core.storage.ManagedItemProviderOSGiTest)  Time elapsed: 0.002 sec
assert tags are stored and retrieve as well(org.eclipse.smarthome.core.storage.ManagedItemProviderOSGiTest)  Time elapsed: 0.003 sec
updating existing item returns old value(org.eclipse.smarthome.core.storage.ManagedItemProviderOSGiTest)  Time elapsed: 0.003 sec
assert remove recursively works(org.eclipse.smarthome.core.storage.ManagedItemProviderOSGiTest)  Time elapsed: 0.004 sec

Results :

Failed tests: 
  AbstractWatchServiceTest.AbstractWatchQueueReader processWatchEvent path in subsubdir provided with correct arguments:91->assertByRelativePath:242->fullEventAssertionsByKind:258->OSGiTest.waitForAssert:181 At least one watch event is expected within 3 seconds.
Expected: is <true>
     but: was <false>
  AbstractWatchServiceTest.directory changes are watched correctly:151->OSGiTest.waitForAssert:181 Exactly 1 watch events were expected within 3 seconds, but were: []
Expected: is <1>
     but: was <0>
  AbstractWatchServiceTest.AbstractWatchQueueReader processWatchEvent path in root provided with correct arguments:75->assertByRelativePath:242->fullEventAssertionsByKind:258->OSGiTest.waitForAssert:181 At least one watch event is expected within 3 seconds.
Expected: is <true>
     but: was <false>
  AbstractWatchServiceTest.same file names in root and subdir are correctly processed:119->OSGiTest.waitForAssert:181 Exactly two watch events were expected within 3 seconds, but were: []
Expected: is <2>
     but: was <0>
  AbstractWatchServiceTest.AbstractWatchQueueReader processWatchEvent path in subdir provided with correct arguments:83->assertByRelativePath:242->fullEventAssertionsByKind:258->OSGiTest.waitForAssert:181 At least one watch event is expected within 3 seconds.
Expected: is <true>
     but: was <false>

Tests run: 182, Failures: 5, Errors: 0, Skipped: 1


```

**Comment by dstivanov** [comment]
Hi,

I took a look and it seems that no events are processed within the tests execution on your machine. Can you run the tests in Eclipse IDE with the ch.qos.logback.classic, ch.qos.logback.core, ch.qos.logback.slf4j and slfj.api bundles included so we can see more logs from the tests executions. 

Thanks,
Dimitar

**Comment by tomhoefer** [comment]
The only relvant output of the tests is:

```
09:37:41.018 [Dir Watcher] DEBUG o.e.s.c.s.AbstractWatchQueueReader - ClosedWatchServiceException catched! null. 
Dir Watcher Stopping 
09:37:44.041 [Dir Watcher] DEBUG o.e.s.c.s.AbstractWatchQueueReader - ClosedWatchServiceException catched! null. 
Dir Watcher Stopping 
09:37:47.050 [Dir Watcher] DEBUG o.e.s.c.s.AbstractWatchQueueReader - ClosedWatchServiceException catched! null. 
Dir Watcher Stopping 
09:37:50.260 [Dir Watcher] DEBUG o.e.s.c.s.AbstractWatchQueueReader - ClosedWatchServiceException catched! null. 
Dir Watcher Stopping 
09:37:53.263 [Dir Watcher] DEBUG o.e.s.c.s.AbstractWatchQueueReader - ClosedWatchServiceException catched! null. 
Dir Watcher Stopping 
09:37:59.271 [Dir Watcher] DEBUG o.e.s.c.s.AbstractWatchQueueReader - ClosedWatchServiceException catched! null. 
Dir Watcher Stopping 
09:38:02.278 [Dir Watcher] DEBUG o.e.s.c.s.AbstractWatchQueueReader - ClosedWatchServiceException catched! null. 
Dir Watcher Stopping 
```
